backend/src/common/util/service.go

```diff
@@ -74,8 +77,19 @@ func GetKubernetesClientFromClientConfig(clientConfig clientcmd.ClientConfig) (
 	return clientSet, config, namespace, nil
 }
 
-func GetRpcConnection(address string) (*grpc.ClientConn, error) {
-	conn, err := grpc.Dial(address, grpc.WithInsecure())
+func GetRpcConnection(address string, tlsEnabled bool) (*grpc.ClientConn, error) {
+	creds := insecure.NewCredentials()
+	if tlsEnabled {
+		config := &tls.Config{
```

Comment:
Nitpick: this code is similar to code [here](https://github.com/opendatahub-io/data-science-pipelines/pull/40/files#diff-d3ae27a3b5003f91b2cbf384575c21f07dd142165daf842fc79fd8459a844829R77). Both instances involve initializing a `tls.Config` structure, setting certs, some error handling. You could create a common function for initializing the TLS configuration instead, to avoid redundancy.

Reply:
these are different procedures doing different things: 

```
GetRpcConnection: builds an rpc connection client given an address/tls config
InitCerts: parsing environment to build a tlsconfig 
```

Creating tls configs is not introducing that much overhead that I think we need a separate procedure for.